tasks/upload-sbom-to-atlas/tests/mocks.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -eux
+
+# mocks to be injected into task step scripts
+curl() {
+  # Output the call to stderr
+  echo "Mock curl called with:" "$@" >&2
+  echo -n "$@" >> "${workdir}/mock_curl.txt"
+  tail -c1 "${workdir}/mock_curl.txt" | read -r _ || printf "\n" >> "${workdir}/mock_curl.txt"
```

Comment:
What's the purpose of this? Why not just print the new line in the echo above (i.e. remove `-n`)?

Reply:
This is leftover from my local testing, thank you for pointing it out!